src/cmds/build/bundle.ts

```diff
@@ -1,37 +1,36 @@
-const { promises: fs, createWriteStream } = require('fs');
-const browserify = require('browserify');
-const stripComments = require('strip-comments');
-// const terser = require('terser')
+import { promises as filesystem, createWriteStream } from 'fs';
+import { logError } from '../../utils';
+import browserify from 'browserify';
+import stripComments from 'strip-comments';
 
-const { logError } = require('../../utils');
-
-module.exports = { bundle };
+/* Custom Type Imports */
```

Comment:
This comment should be removed, because it's not communicating anything that isn't immediately apparent from the path of the imports it's explaining.

The same goes for wherever this comment was added!